src/core/Tooltip/Tooltip.tsx

```diff
@@ -22,7 +22,8 @@ export type TooltipProps = {
   Omit<CommonProps, 'title'>;
 
 /**
- * Basic tooltip component
+ * Basic tooltip component to display informative content when an element is hovered or focused.
+ * Uses the Popover component, which is a wrapper around {@link https://atomiks.github.io/tippyjs tippy.js}.
```

Comment:
Damnit, VSCode 1.57 broke this nice feature
![image](https://user-images.githubusercontent.com/9084735/122464827-4b8c0680-cf85-11eb-95e3-5d9ce765b918.png)

Reply:
I see it's fixed in https://github.com/microsoft/vscode/commit/7339e948305024dbf0479e926e7e5bccb8baa97b but I think that might take a while to release.

Meanwhile I've replaced those tags with markdown syntax.